doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,267 @@
+.. _setup_logical_partition_scenario_on_nuc7i7dnb:
+
+Guidelines of Setup Logical Partition Scenario on NUC7i7DNB
```

Comment:
@dbkinder Sorry about this name confuses you.  "NUC7i7DNHE" is listed on the https://projectacrn.github.io/latest/hardware.html#intel-kaby-lake-nuc. As we known, the intel KBL NUC has product name and board name. The platform is used here, "NUC7i7DNHE" is its product name, "NUC7i7DNB" is its board name. BTW, in fact, the board "NUC7i7DNB" is supported, there are three products for this board, please refer to page #iii of https://www.intel.com/content/dam/support/us/en/documents/mini-pcs/nuc-kits/NUC7i7DN_TechProdSpec.pdf. the board name may be precise for developer and customer. Do you have some suggestion about which name is needed to be used? Thanks!

Reply:
The board name of  "Intel NUC Kit NUC7i7DNHE" on the hw page is “NUC7i7DNB”, can we use  "Intel NUC Kit NUC7i7DNHE" instead of NUC7i7DNB? or do you have another suggestion about this, thanks!